tests/system/python/packages/test_authentication.py

```diff
@@ -0,0 +1,1157 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test authentication REST API """
+
+import os
+import subprocess
+import http.client
+import json
+import time
+import pytest
+from pathlib import Path
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+PROJECT_ROOT = ""
+SCRIPTS_DIR_ROOT = ""
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 12.25
+# TODO : pass package_build_version to setup script from conftest.py
+package_build_version = "nightly"
+HTTP_SOUTH_SVC_NAME = "SOUTH_HTTP"
+HTTP_SOUTH_SVC_NAME_1 = "SOUTH_HTTP_1"
+ASSET_NAME = "auth"
+PASSWORD_TOKEN = None
+CERT_TOKEN = None
+
+
+@pytest.fixture
+def set_project_root():
+    global PROJECT_ROOT, SCRIPTS_DIR_ROOT
```

Comment:
No need to define these as global (and using a def), but module level constants.